src/marketing/prex-type-trending.md

```diff
@@ -0,0 +1,34 @@
+---
+ee_only: true
+title: Trending
+---
+
+The _Trending_ recommendation type recommends items based on recent momentum of a product’s popularity across your site. Trending products are those often referred to as "Trending now", "Hot items", and so on.
+
+## How it works
+
+Adobe Sensei aggregates browsing and purchase data across your site to determine and rank which products are the most recently popular with your shoppers. Because _Trending_ analyzes recent product momentum, it is an effective recommendation type for catalogs that have a high turnover. If your catalog is more static, it might not be as useful as the data used to evaluate a product's momentum is not available.
```

Comment:
```suggestion
Adobe Sensei aggregates browsing and purchase data across your site to determine and rank which products are the most recently popular with your shoppers. Because _Trending_ analyzes recent product momentum, it is an effective recommendation type for catalogs that have a high turnover. If your catalog is more static, it might not be as useful unless the shopping patterns of your audience are highly variable.
```